test/remote/gateways/remote_plexo_test.rb

```diff
@@ -158,4 +158,103 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.number, transcript)
     assert_scrubbed(@gateway.options[:api_key], transcript)
   end
+
+  def test_successful_purchase_passcard
+    credit_card = credit_card('6280260025383009', month: '12', year: '2024',
```

Comment:
💬  Comment:

Small thing => just to prevent silly errors on the future maybe a year value like `Time.now.year + 2` is a good idea